misc/docker-ci/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# To build Docker images for CI.
+# https://hub.docker.com/r/h2oserver/h2o-ci
+
+set -xe
+
+IMAGE_NAME=h2oserver/h2o-ci
+REVS="ubuntu1604 ubuntu2004"
+
+PROJECT_ROOT_DIR=$(realpath "$(dirname $0)/../..")
+cd $PROJECT_ROOT_DIR
+
+docker login
+
+for rev  in $REVS ; do
```

Comment:
Instead of iterating through the hard-coded revs, how about taking the rev to be built from the argument of the command?

As an example, `misc/docker-ci/build.sh ubuntu2004` could build h2o-ci:ubuntu2004 from misc/docker-ci/Dockerfile.ubuntu2004 and push the image being built. The rationale is that we do not need to hard code the revs, and that we do not need to push both images when making change to just one of those Dockerfiles.